website/docs/docs/building-a-dbt-project/building-models/configuring-incremental-models.md

```diff
@@ -64,14 +67,27 @@ from raw_app_data.events
 
 :::tip Optimizing your incremental model
 
-For more complex incremental models that make use of CTEs, you should consider the impact of the position of the `is_incremental()` macro on query performance. On some warehouses, filtering your records early can vastly improve the run time of your query!
+For more complex incremental models that make use of Common Table Expressions (CTEs), you should consider the impact of the position of the `is_incremental()` macro on query performance. In some warehouses, filtering your records early can vastly improve the run time of your query!
 
 :::
 
 ### Defining a uniqueness constraint (optional)
-`unique_key` is an optional parameter for incremental models that specifies a field which should be unique within your model. If the unique key of an existing row in your target table matches one of your incrementally transformed rows, the existing row will be updated. This ensures that you don't have multiple rows in your target table for a single row in your source data.
 
-You can define `unique_key` in a configuration block at the top of your model. The `unique_key` should be a single field name that is present in your model definition. While some databases support using expressions (eg. `concat(user_id, session_number)`), this syntax is not universally supported, so is not recommended. If you do not have a single field that is unique, consider first creating such a field in your model.
+A `unique_key` is an optional parameter for incremental models that specifies a field that should be unique within your model. You can define `unique_key` in a configuration block at the top of your model.
```

Comment:
Given the change in v1.1, how about something like:
```suggestion
A `unique_key` is an optional parameter for incremental models, that specifies a field (or combination of fields) that can uniquely identify each row within your model. You can define `unique_key` in a configuration block at the top of your model.
```
We may want to `VersionBlock` that language, and keep the previous language for older versions

Reply:
Addressed in https://github.com/dbt-labs/docs.getdbt.com/pull/1298/commits/0de7fe73136f06ff9b94ba4152a38897c6e407dd so I could add versioning as well.